### DISCUSSION TITLE
How to detect exact degree of a rotate text/table image? 0-45, 45-90,95-135,135-180, 180-225, 215-270, 315-360 degrees etc.

### DISCUSSION THREAD

**Discussion by nissansz** [python-pillow/Pillow#7919] [Q&A] [answered]
How to detect exact degree of a rotate text/table image? 0-45, 45-90,95-135,135-180, 180-225, 215-270, 315-360 degrees etc.

**Comment by radarhere** [comment, marked as answer]
You have already asked about detecting tables in images (#7370) and detecting text in images (#7376).

If Pillow doesn't have any particular tools for detecting tables or text, then it won't have any tools for detecting to what degree tables or text are rotated.

**Comment by nissansz** [comment]
ok
